app/app.py

```diff
@@ -58,11 +58,12 @@ def cosmos_client() -> "CosmosClient":
     Documentation: https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/sdk-python
     """
     from azure.cosmos import CosmosClient
-    from azure.identity import DefaultAzureCredential
+    from azure.identity import AzureCliCredential
+
 
     client = CosmosClient(
         os.environ["COSMOSDB_ENDPOINT"],
-        credential=(DefaultAzureCredential() if environment != "local"
+        credential=(AzureCliCredential() if environment != "local"
```

Comment:
Don't think you want this change in the PR

Reply:
Resolved 81a8ae7c1efe4ccc63e6b2af2537cf4ec8bf1ef8